manim/default.cfg

```diff
@@ -156,6 +149,16 @@ streaming_console_banner = Manim is now running in streaming mode.
     >>> c = Circle()
     >>> manim.play(ShowCreation(c))
 
+# Overrides the default output folders, NOT the output file names.  Note that
+# if the custom_folders flag is present, the Tex and text files will not be put
+# under media_dir, as is the default.
+[custom_folders]
+media_dir = videos
+video_dir = %(media_dir)s
```

Comment:
Can we maybe add here as well the option
video_dir = %(media_dir)s ?
Cause at the moment a video would go to:
`videos/TestExample.mp4`
but an image would go to:
`videos/images/basic/TestExample.png`
it would be nice to have instead:
`videos/TestExample.png`

Reply:
Done in the last commit.